clients/java/client/src/test/java/org/camunda/bpm/client/impl/ExternalTaskClientBuilderImplTest.java

```diff
@@ -91,4 +94,17 @@ public String getBaseUrl() {
     }
   }
 
+  @Test
+  public void testWithNullBaseUrl() {
+    // given
+    ExternalTaskClientBuilderImpl builder = new ExternalTaskClientBuilderImpl();
+
+    // when building with null base url then exception is thrown
```

Comment:
I think this comment can be misleading about the purpose of the test case, how about something like
```suggestion
    // when retrieving baseURL before calling build, no NPE is thrown
```